ammico/cropposts.py

```diff
@@ -1,58 +1,51 @@
 import os
 import ntpath
-from PIL import Image
-from matplotlib.patches import ConnectionPatch
 import cv2
 import pandas as pd
 import numpy as np
 import matplotlib.pyplot as plt
+from ammico import utils
+from typing import List, Optional, Tuple, Union
 
 
-# use this function to visualize the matches
-def plot_matches(img1, img2, keypoints1, keypoints2):
-    fig, axes = plt.subplots(1, 2, figsize=(16, 7))
-
-    # draw images
-    axes[0].imshow(img1)
-    axes[1].imshow(img2)
-
-    # draw matches
-    for kp1, kp2 in zip(keypoints1, keypoints2):
-        c = np.random.rand(3)
-        con = ConnectionPatch(
-            xyA=kp1,
-            coordsA=axes[0].transData,
-            xyB=kp2,
-            coordsB=axes[1].transData,
-            color=c,
-        )
-        fig.add_artist(con)
-        axes[0].plot(*kp1, color=c, marker="x")
-        axes[1].plot(*kp2, color=c, marker="x")
-
-    plt.show()
+MIN_MATCH_COUNT = 6

```

Comment:
In `draw_matches` function `MIN_MATCH_COUNT` resets to 4, but in `crop_posts_image` it uses `MIN_MATCH_COUNT=6`. Is it supposed to be like this?